pyproject.toml

```diff
@@ -1,2 +1,7 @@
 [tool.black]
-line-length = 79
+exclude = '/(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv.*|_build|buck-out|build|dist)/'
+line-length = 88
+
+[tool.isort]
+profile = "black"
+skip_glob = '.venv*'
```

Comment:
Careful about new-line at end of file

Reply:
I don't think it matters here. It is this way in PRAW's pyproject.toml file.